x/poolmanager/client/query_proto_wrap.go

```diff
@@ -214,6 +214,30 @@ func (q Querier) AllPools(ctx sdk.Context, req queryproto.AllPoolsRequest) (*que
 	}, nil
 }
 
+// ListPoolsByDenom return list pools by denom
+func (q Querier) ListPoolsByDenom(ctx sdk.Context, req queryproto.ListPoolsByDenomRequest) (*queryproto.ListPoolsByDenomResponse, error) {
+	if req.Denom == "" {
+		return nil, status.Error(codes.InvalidArgument, "invalid quote asset denom")
```

Comment:
why does it have to be quote asset?
```suggestion
		return nil, status.Error(codes.InvalidArgument, "invalid denom")
```